README.md

```diff
@@ -18,7 +18,100 @@ MiNiFi is a child project effort of Apache NiFi
 
 ## Table of Contents
 
+- [Features](#features)
+- [Requirements](#requirements)
+- [Getting Started](#getting-started)
+- [Getting Help](#getting-help)
+- [Documentation](#documentation)
 - [License](#license)
+- [Export Control](#export-control)
+
+## Features
+
+Apache NiFi - MiNiFi is a complementary data collection approach that supplements the core tenets of [NiFi](http://nifi.apache.org/) in dataflow management, focusing on the collection of data at the source of its creation.
+
+Specific goals for MiNiFi are comprised of:
+- small and lightweight footprint
+- central management of agents
+- generation of data provenance
+- integration with NiFi for follow-on dataflow management and full chain of custody of information
+
+Perspectives of the role of MiNiFi should be from the perspective of the agent acting immediately at, or directly adjacent to, source sensors, systems, or servers.
+
+## Requirements
+* JDK 1.8 or higher
+* Apache Maven 3.1.0 or higher
+
+## Getting Started
+
+- Read through the [quickstart guide for development](http://nifi.apache.org/quickstart.html).
```

Comment:
Agreed, will remove this for the time being and have created a ticket to improve MINIFI-51.